pontoon/base/static/js/translate.js

```diff
@@ -429,7 +429,21 @@ var Pontoon = (function (my) {
      * Update current translation length
      */
     updateCurrentTranslationLength: function () {
-      $('#translation-length .current-length').html($('#translation').val().length);
+      var limit = this.translationLengthLimit,
+          translation = $('#translation').val();
+
+      if (limit) {
+        var length = this.stripHTML(translation).length,
```

Comment:
[The spec](https://github.com/mozilla-l10n/langchecker/wiki/.lang-files-format#string-meta-data) doesn't specify anything.

BTW, this is Mozilla's format, probably nobody else uses it, so it's a little easier to make some asumptions. :P